autogpt_platform/backend/schema.prisma

```diff
@@ -19,45 +19,164 @@ model User {
   updatedAt DateTime @updatedAt
 
   // Relations
-  AgentGraphs                  AgentGraph[]
+  AgentGraphs                  Agent[]
   AgentGraphExecutions         AgentGraphExecution[]
   AgentGraphExecutionSchedules AgentGraphExecutionSchedule[]
```

Comment:
Similarly to what [Pwuts mentioned](https://github.com/Significant-Gravitas/AutoGPT/pull/8375/files#r1806232014), change from `AgentGraph` to `Agent` makes this schema inconsistent; why isn't `AgentGraphExecutions*`, `AgentGraphVersion`, etc. not renamed?

Reply:
Updated